src/convertToObject.js

```diff
@@ -1,12 +1,27 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const linesOfString = sourceString
+    .split(';')
+    .reduce((accumulator, line) => {
+      if (line.includes(':')) {
+        const lineProperties = line
+          .split(':')
+          .map((property) => property.trim());
+
+        accumulator.push(lineProperties);
+      }
+
+      return accumulator;
```

Comment:
Everything is good, but chaining contributes to the worse readability of the code. Try splitting the code into constants.
Also, you can try to use one reduce method.